src/redux/configureStore.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ]  I would recommend not to disable linter rules. These are here to help us write clean code.